chain/client/tests/catching_up.rs

```diff
@@ -875,15 +867,15 @@ mod tests {
                             header.inner.shard_id,
                         ));
                     }
-                    if let NetworkRequests::PartialEncodedChunkRequest { account_id, request } = msg
+                    if let NetworkRequests::PartialEncodedChunkRequest { account_id: _, request } =
+                        msg
                     {
-                        println!("S {:?} R {:?} PEQ {:?}", sender_account_id, account_id, request);
                         if requested.contains(&(
                             sender_account_id.clone(),
                             request.part_ords.clone(),
                             request.chunk_hash.clone(),
                         )) {
-                            println!("== SAME REQUEST AGAIN!");
+                            // do nothing
```

Comment:
Yeah, I think this looks even worse :)

We should either completely remove these conditions, or, if you foresee us debugging here something again, reintroduce tracing but add a `verbose` flag at the beginning of the test, and then change these snippets to:

```
if vebose && requested.contains(&(...
 ...
{
     println!("== SAME REQUEST AGAIN!");
```